portality/lib/es_queries.py

```diff
@@ -0,0 +1,13 @@
+ES_DATETIME_FMT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+def query_all():
+    return {
+        'query': {
+            'bool': {
+                'must': [
+                    {'match_all': {}}
+                ]
+            }
+        }
+    }
```

Comment:
The match all query can be written a bit simpler as

```
{
    "query": {
        "match_all": {}
    }
}
```

You only need the `bool.must` bit if you are (potentially) going to have some other constraints too